airflow/www/views.py

```diff
@@ -459,6 +459,97 @@ def add_user_permissions_to_dag(sender, template, context, **extra):  # noqa pyl
         context['dag'] = dag
 
 
+def get_tree_data(dag_runs, dag, base_date):
```

Comment:
```suggestion
def get_tree_data(dag_runs: Iterable[DagRun], dag: DAG, base_date: DateTime):
```

(I think those types should be in scope and correct)